apps/music/js/ui/views/subList_view.js

```diff
@@ -139,12 +143,15 @@ var SubListView = {
     }.bind(this));
   },
 
-  update: function slv_update(result) {
+  // Set inPlaylist to true if you want the index instead of the track #
+  // By default it is the track #
+  update: function slv_update(result, inPlaylist) {
```

Comment:
I'd rename `inPlaylist` to `useIndexNumber` or something like that. That'd make it clearer what this argument does.